src/ducks/categories/CategoryChoice.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import sortBy from 'lodash/sortBy'
-import Fuse from 'fuse.js'
+import Fuse from 'fuse.js/dist/fuse.js'
```

Comment:
you should import the minified version: 

```
➜  fuse.js git:(master) ✗ ls -la dist
total 744
-rw-r--r--   1 quentinvalmori  staff  41793 10 jui 14:59 fuse.basic.common.js
-rw-r--r--   1 quentinvalmori  staff  28864 10 jui 14:59 fuse.basic.esm.js
-rw-r--r--   1 quentinvalmori  staff  10013 10 jui 14:59 fuse.basic.esm.min.js
-rw-r--r--   1 quentinvalmori  staff  44470 10 jui 14:59 fuse.basic.js
-rw-r--r--   1 quentinvalmori  staff  14607 10 jui 14:59 fuse.basic.min.js
-rw-r--r--   1 quentinvalmori  staff  62512 10 jui 14:59 fuse.common.js
-rw-r--r--   1 quentinvalmori  staff   6894 10 jui 14:59 fuse.d.ts
-rw-r--r--   1 quentinvalmori  staff  41031 10 jui 14:59 fuse.esm.js
-rw-r--r--   1 quentinvalmori  staff  15439 10 jui 14:59 fuse.esm.min.js
-rw-r--r--   1 quentinvalmori  staff  66491 10 jui 14:59 fuse.js
-rw-r--r--   1 quentinvalmori  staff  22871 10 jui 14:59 fuse.min.js
```

Reply:
Why ? it will be modified in the build.

Reply:
A link to the fuse's issue can be useful to know when to remove this import